mlrun/datastore/targets.py

```diff
@@ -106,6 +106,19 @@ def add_target_states(graph, resource, targets, to_df=False, final_state=None):
     return table
 
 
+def _get_column_list(features, timestamp_key, key_columns):
+    column_list = None
+    if features:
+        column_list = list(features.keys())
+        if timestamp_key and timestamp_key not in column_list:
+            column_list = [timestamp_key] + column_list
+        if key_columns:
+            for key in reversed(key_columns):
```

Comment:
why r we using `reversed`? also do we need to add the index key in all targets?

Reply:
Both are existing logic I just wanted to preserve in my refactoring.
reversed isn't used in Parquet & the timestamp key isn't used in TSDB.